apps/emqx/include/emqx_placeholder.hrl

```diff
@@ -41,94 +41,94 @@
 
 -define(PH_PASSWORD, ?PH(?VAR_PASSWORD)).
 -define(PH_CLIENTID, ?PH(?VAR_CLIENTID)).
--define(PH_FROM_CLIENTID, ?PH("from_clientid")).
+% -define(PH_FROM_CLIENTID, ?PH("from_clientid")).
```

Comment:
the PH_ macros are used in tests.
maybe warp them around `-ifdef(TEST)`.

Reply:
I've just disabled the rule here and added a comment to justify!